packages/h5wasm/src/utils.ts

```diff
@@ -34,3 +37,35 @@ export function assertH5WasmEntityWithAttrs(
     throw new Error('Expected H5Wasm entity with attributes');
   }
 }
+
+export function convertMetadataToDType(metadata: Metadata): DType {
+  const { signed, type, size: length, littleEndian } = metadata;
+
+  switch (type) {
```

Comment:
https://github.com/usnistgov/h5wasm/blob/2a006aad8d4c825e6193dd02540c89bbbe5a6457/src/hdf5_util_helpers.d.ts#L5

Reply:
Maybe put a link to https://github.com/usnistgov/h5wasm/blob/main/src/hdf5_util_helpers.d.ts in a comment?